### PR TITLE
ENH: update grammar for py39

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

**Comment by davidhalter** [comment]
Please just copy the 3.8 grammar (from `parso/python/`) for now and put it there. I will review it anyway from time to time. There's a couple of changes from the official grammar now, so I cannot just use this one.

**Comment by tacaswell** [comment]
Sorry, I don't understand what you are asking me to do :/

I was trying to do the same thing I did in https://github.com/davidhalter/parso/pull/52

**Comment by davidhalter** [comment]
Please just copy the grammar from `parso/python/grammar38.txt` to `parso/python/grammar39.txt`.

**Comment by tacaswell** [comment]
ok, they are now the same, but I left revertible commits to add things back ;)

File: parso/python/token.py [hunk 1]
```diff
@@ -22,6 +22,6 @@ def __init__(self, names, contains_syntax):
 PythonTokenTypes = TokenTypes((
     'STRING', 'NUMBER', 'NAME', 'ERRORTOKEN', 'NEWLINE', 'INDENT', 'DEDENT',
     'ERROR_DEDENT', 'FSTRING_STRING', 'FSTRING_START', 'FSTRING_END', 'OP',
-    'ENDMARKER'),
+    'ENDMARKER', 'ASYNC', 'TYPE_COMMENT', 'AWAIT'),
```
Comment (davidhalter) [inline parso/python/token.py]: Please remove this, it's not going to be part of parso.

**Comment by tacaswell** [comment]
Done.  I do not really understand why you want to maintain the divergence from upstream, but I also do not understand the internals of `parso` or `jedi`.

**Comment by davidhalter** [comment]
There's a couple differences that I don't want to deal with in Jedi and parso.

Also I use a grammar for f-strings (and not reparsing of parts of the tree).

**Comment by tacaswell** [comment]
Fair enough.  

Thanks!  I can go back to using the parso master branch now :)

**Pull request merged**